### ISSUE TITLE
Potential security vulnerability

### ISSUE THREAD

**Issue by Dostoyewski** [daniilmir3105/first_work#1]
* You left a secret key in the public domain. And uploaded the database;
* This doesn't apply to security, but still: gitignore setup.

**Comment by daniilmir3105** [comment]
I completely agree with the first point and found a solution for this problem.
I don't understand second point. Can you write about this problem more concretely.

**Comment by Dostoyewski** [comment]
Yes, of course. I saw that you already added gitignore. If I were you, I'd add a few more lines. See gitignore configuration for django.
